plugins/themes/default/styles/objects/issue_summary.less

```diff
@@ -10,6 +10,11 @@
  */
 .obj_issue_summary {
 
+	h2 {
+		margin: 0;
+		font-size: @font-lead;
+	}
```

Comment:
It looks like before these changes the issue titles were smaller:

![Selection_172](https://user-images.githubusercontent.com/2306629/70633668-d1a8ed80-1c28-11ea-8d44-cd21dddfda74.png)

I like your font sizing here, but let's try to keep it the same as it was to minimise disruption for people who may be using a custom child theme.

So change `font-size` to `@font-base`. Also, the `h2` will effect the `line-height` as well, so add a `line-height` prop and set it to `@line-base`.